src/styles/main.scss

```
@@ -1,3 +1,51 @@
+$height: 300px;
+
+html {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 100px;
+  color: white;
+}
+
 body {
   margin: 0;
 }
+
+.wrap {
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax(300px, 1fr));
+
+  @media (min-width: 900px) {
+    grid-template-columns: repeat(3, 1fr);
+  }
+}
+
+.container {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: $height;
+
+  &--1 {
+    background-color: rgb(255, 0, 0);
```

you can use percentages here
```suggestion
    background-color: rgb(100%, 0, 0);
```